tagging/model.py

```diff
@@ -37,7 +41,16 @@ class Config:
         extra = Extra.forbid
 
 
-class Tag(BaseModel):
+class UserTag(BaseModel):
```

Comment:
This would be cleaner if `Tag` inherited from `UserTag`...then you wouldn't have to repeat (and maintain) the fields in the two different classes. Tag would just become:

Also, I'm starting to thing that the name `UserTag`is maybe a little misleading.

What do you think of this?

```
def Tag(BaseModel):
  ... some fields  ...

def PersistedTag(Tag, Persistable):
  pass

```